frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/Job.java

```diff
@@ -72,9 +73,17 @@ public void setScheduled() {
         scheduled = System.currentTimeMillis();
     }
 
-    public void response(byte[] body, CharSequence contentType) {
+    public void response(
+            byte[] body, CharSequence contentType, int statusCode, String statusPhrase) {
         FullHttpResponse resp =
-                new DefaultFullHttpResponse(HttpVersion.HTTP_1_1, HttpResponseStatus.OK, false);
+                new DefaultFullHttpResponse(
```

Comment:
HttpResponseStatus.valueOf() will create a HttPResponseStatus object, which is not necessary:

            HttpResponseStatus status;
            if (reasonPhrase == null) {
                status = HttpResponseStatus.valueOf(statusCode);
            } else {
                status = HttpResponseStatus.valueOf(statusCode, reasonPhrase);
            }

Reply:
Done.